onthequad/onthequad/SignInEndPoints.py

```diff
@@ -0,0 +1,26 @@
+import falcon
+import json
+from .User import User
+from .UsersDB import UsersDB
+from .DataBaseSetUp import DataBaseSetUp
+authen = DataBaseSetUp.authentication()
+
+class SignInEndPoints:
+
+    def on_post(self, req, resp):
+        data = json.loads(req.stream.read())
```

Comment:
I added some logic to check if there is any data, the field is valid or not (it has to be email and password, otherwise throughout an error message ), then check for credentials.